system/cfme-setup.sh

```diff
@@ -31,6 +31,17 @@ cat <<'EOF' > /etc/httpd/conf.d/ssl.conf
 # upgraded.
 EOF
 
+# Incorporate the iptable updates
+if [[ -f /etc/sysconfig/iptables.miq ]]; then
+  /sbin/iptables-save > /etc/sysconfig/iptables.SYSTEM
+  /sbin/iptables-restore /etc/sysconfig/iptables.miq
+  /sbin/service iptables save
+  # Expect no more than the four datestamp differences.
+  if [[ $(/usr/bin/diff -Ew -U 0 /etc/sysconfig/iptables /etc/sysconfig/iptables.miq  | /bin/grep "^@@" | /usr/bin/wc -l) -gt 4 ]]; then
```

Comment:
@jrafanie  Thanks for the feedback.

Personally I don't really see any harm in logging a message if a failure is detected.
It could come in handy and I don't see and downsides to leaving it in.

I added this only because you had mentioned checking it in our discussion.

That said I'm OK with whatever you think is best.